Add more space below the Rust badge in the README

camelid opened the pull request:
The extra space makes it easier to read the README.

rust-highfive commented:
r? @nikomatsakis

(rust_highfive has picked a reviewer for you, use r? to override)

jyn514 commented:
r? @steveklabnik

steveklabnik commented:
I am not sure how I feel about adding an increasing amount of literal HTML in our markdown. I'm not going to block this, but I'm not sure that I think it should be merged either.

camelid commented:
I understand where you're coming from, I don't love all the HTML either. It just feels cramped to me right now, so that's why I wanted a bit more space:

![image](https://user-images.githubusercontent.com/37223377/91591232-77b09b00-e911-11ea-8aee-d23e9b0413fb.png)

camelid commented on README.md:
```diff
@@ -1,6 +1,8 @@
+<p>
 <a href = "https://www.rust-lang.org/">
 <img width = "90%" height = "auto" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust" alt = "The Rust Programming Language">
 </a>
+</p>
 
```
If you don't want to add more HTML, what about a horizontal line? Like this:

```suggestion
<a href = "https://www.rust-lang.org/">
<img width = "90%" height = "auto" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust" alt = "The Rust Programming Language">
</a>

---

```

camelid commented:
I changed it to add a horizontal line (in Markdown) instead and I rebased atop master. I can close this if you still would rather not have it, but I think it makes it more readable.

camelid commented:
I would actually prefer just a plain-text title (like how it used to be), but I figured other people liked it better the way it looks now.

Dylan-DPC commented:
Closing this as there doesn't seem to be motivation to take this further from the author and the reviewer.
